docs/source/user/extptfm/theory.rst

```diff
@@ -0,0 +1,514 @@
+transformation:
+:math:`\boldsymbol{M}_r =\boldsymbol{T}^t \boldsymbol{M} \boldsymbol{T}`,
+:math:`\boldsymbol{K}_r =\boldsymbol{T}^t \boldsymbol{K} \boldsymbol{T}`,
+:math:`\boldsymbol{f}_r =\boldsymbol{T}^t \boldsymbol{f}`,
+leading to
+:math:`\boldsymbol{M}_r \boldsymbol{\ddot{x}}_r + \boldsymbol{K_r}\boldsymbol{x}_r=\boldsymbol{f}_r`,
+which is written in a developed form as:
+
+.. math:: :label: CBLoadsReduction
+
+   \begin{aligned}
+       &
+           \qquad
+           \qquad
+           \begin{bmatrix}
+           \boldsymbol{M}_{r11} & \boldsymbol{M}_{r12} \\
+           \boldsymbol{M}_{r12}^t & \boldsymbol{M}_{r22} \\
+       \end{bmatrix}
+       \begin{bmatrix}
+           \boldsymbol{\ddot{x}}_{r1}\\
+           \boldsymbol{\ddot{x}}_{r2}\\
+       \end{bmatrix}
+       +
+       \begin{bmatrix}
+           \boldsymbol{K}_{r11} & \boldsymbol{0} \\
+       \boldsymbol{0} & \boldsymbol{K}_{r22} \\
+       \end{bmatrix}
+       \begin{bmatrix}
+           \boldsymbol{x}_{r1}\\
+           \boldsymbol{x}_{r2}\\
+       \end{bmatrix}
+       =
+       \begin{bmatrix} 
+           \boldsymbol{f}_{r1}\\
+           \boldsymbol{f}_{r2}\\
+       \end{bmatrix}
+       \label{eq:CraigBampton}
+   \end{aligned}
+
+with
+
+.. math::
+
+   \begin{aligned}
+      & \boldsymbol{M}_{r11} = \boldsymbol{M}_{{\ell\!\ell}}
+                     + \boldsymbol{\Phi}_\text{1}^t \boldsymbol{M}_{f\!\ell}
+                     + \boldsymbol{M}_{\ell\!f}\boldsymbol{\Phi}_\text{1}
+                     + \boldsymbol{\Phi}_\text{1}^t \boldsymbol{M}_{\!f\!f}\boldsymbol{\Phi}_{1}
+   ,\qquad
+       \boldsymbol{M}_{r22}=\boldsymbol{\Phi}_2^t\boldsymbol{M}_{\!f\!f}\boldsymbol{\Phi}_2 =\boldsymbol{I}
+   \nonumber
+   \\
+      & \boldsymbol{M}_{r12} = \left(\boldsymbol{M}_{\ell\!f}+ \boldsymbol{\Phi}_\text{1}^t \boldsymbol{M}_{\!f\!f}\right)\boldsymbol{\Phi}_2
+   ,\qquad
+       \boldsymbol{f}_{r2} = \boldsymbol{\Phi}_2^t \boldsymbol{f}_{\!f}
+   ,\qquad
+       \boldsymbol{f}_{r1} = \boldsymbol{f}_{\ell}+\boldsymbol{\Phi}_1^t \boldsymbol{f}_{\!f}
+   \nonumber
+   \\
+      & \boldsymbol{K}_{r11} = \boldsymbol{K}_{{\ell\!\ell}} + \boldsymbol{K}_{\ell\!f}\boldsymbol{\Phi}_1
+   ,\qquad
+       \boldsymbol{K}_{r22}=\boldsymbol{\Phi}_2^t\boldsymbol{K}_{\!f\!f}\boldsymbol{\Phi}_2
+       \nonumber
+   \end{aligned}
+
+The expressions for the reduced damping matrix,
+:math:`\boldsymbol{C}_r=\boldsymbol{T}^t \boldsymbol{C} \boldsymbol{T}`,
+are similar to the ones from the mass matrix, except that
+:math:`\boldsymbol{C}_{r22}` is not equal to the identity matrix. Some
+tools or practitioners may not compute the reduced damping matrix and
+instead set it based on the Rayleigh damping assumption, using the
+reduced mass and stiffness matrix. Setting
+:math:`\boldsymbol{\Phi}_2\equiv 0` in Eq. :eq:`CraigBampton`, or equivalently
+:math:`n_\text{CB}\equiv 0`, leads to the Guyan reduction equations.
+
+Coupling with another structure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
I suggest clarifying that the "Modular approach" is what is implemented in OpenFAST and the "Monolithic approach" is just useful background information.